### DISCUSSION TITLE
Downloads per week counter is broken

### DISCUSSION THREAD

**Discussion by kelian9** [community/community#137541] [npm]
I checked my little project (https://www.npmjs.com/package/dizy) and noticed the downloads per week had fallen from something like 300 to 0 (0 downloads of all versions!). Why did this happen?

**Comment by CarlosNZ** [comment]
Indeed! I think they stopped counting a few days ago, as I noticed my count plummeting over the last week. Now they're all showing "0", even the the really big projects like https://www.npmjs.com/package/react

**Reply by FreTimmerman** [reply]
they didn't really stop counting. if you hover the graph, it still shows correct number up until [yesterday]
i think the problem is that they don't include [today] in the numbers anymore, and since they don't count [today] then the summary shows 0.

i think their fix will be to show the summary as the count of [yesterday] instead of [today]

**Reply by kelian9** [reply]
Before today, my package’s downloads for the entire time were reset to zero and, accordingly, the weekly downloads graph was not even rendered. Now it's fixed, and the information about downloads has returned, but the number does not correspond to the previous value.

**Reply by CarlosNZ** [reply]
Seems to be back to normal now 🤷‍♂️

**Reply by kelian9** [reply]
Yeah, it works now. There was a similar problem before, in 2018 (https://status.npmjs.org/incidents/b4ktc38pypd1) . I hope this doesn’t happen again)

**Comment by Ebrahim-Ramadan** [comment]
![image](https://github.com/user-attachments/assets/4799ed64-f1bf-47cc-8424-96bee9abec9e)

damn
